otter/test/utils.py

```diff
@@ -846,6 +846,34 @@ def noop(_):
     pass
 
 
+def intent_func(fname):
+    """
+    Return function that returns Effect of tuple of fname and its args. Useful
+    in writing tests that expect intent based on args
+    """
+    return lambda *a: Effect((fname,) + a)
+
+
+class Cache(object):
```

Comment:
this class should probably have a more specific name than Cache. and maybe an `@implements`